package/MDAnalysis/core/universe.py

```diff
@@ -387,11 +387,11 @@ def load_new(self, filename, format=None, in_memory=False, **kwargs):
                                  trj_n_atoms=self.trajectory.n_atoms))
 
         if in_memory:
-            self.transfer_to_memory(kwargs.get("in_memory_frame_interval", 1))
+            self.transfer_to_memory(step=kwargs.get("in_memory_step", 1))
 
         return filename, self.trajectory.format
 
-    def transfer_to_memory(self, frame_interval=1, verbose=None, quiet=None):
+    def transfer_to_memory(self, start=None, stop=None, step=1, verbose=None, quiet=None):
```

Comment:
Step needs to be None here,  other wise with start=10, stop=1 (backwards iteration) it won't work